src/Common/src/TypeSystem/Interop/IL/Marshaller.cs

```diff
@@ -1889,9 +1406,9 @@ protected override void EmitMarshalArgumentManagedToNative()
             }
 
             // we don't support [IN,OUT] together yet, either IN or OUT
-            Debug.Assert(!(PInvokeParameterMetadata.Out && PInvokeParameterMetadata.In));
+            // Debug.Assert(!(PInvokeParameterMetadata.Out && PInvokeParameterMetadata.In));
```

Comment:
There should be a comment explaining why a piece of code is checked in commented out.